pkg/autoscaler/metrics/stats_scraper.go

```diff
@@ -117,7 +118,7 @@ type ServiceScraper struct {
 
 // NewServiceScraper creates a new StatsScraper for the Revision which
 // the given Metric is responsible for.
-func NewServiceScraper(metric *av1alpha1.Metric, counter resources.EndpointsCounter) (*ServiceScraper, error) {
+func NewServiceScraper(metric *av1alpha1.Metric, counter resources.EndpointsCounter) (StatsScraper, error) {
```

Comment:
Since we are making `ServiceScarper` private, should we also change this to `NewStatsScraper`?

Reply:
Sure.